docs/user_guide/usage_with_python.rst

```diff
@@ -64,3 +150,6 @@ directory):
 
 Details about parameters for the various adapters, and differences
 between OSes are covered in the "Adapter Specifics" sections.
+
+If you decided to install Lewis this way, please be aware that the ``lewis`` and ``lewis-control``
+calls in the other parts of the guide have to be replaced with ``python lewis.py``.
```

Comment:
You could also just execute them directly (`$ path/to/lewis.py ...`) since they have a shebang and +x permissions. 

Not sure if running it that way works on Windows though? If it doesn't, it's probably not worth changing anything here. `python lewis.py` will definitely work cross-platform.